src/ConfigWidgets/include/ConfigWidgets/SymbolsDialog.h

```diff
@@ -60,6 +62,7 @@ class SymbolsDialog : public QDialog {
   // Returns the caption and file filter for the FileDialog that is opened when the user clicks the
   // "Add File" button.
   [[nodiscard]] std::tuple<QString, QString> GetFilePickerConfig() const;
+  [[nodiscard]] OverrideWarningResult DisplayOverrideWarning();
```

Comment:
`const`?

Reply:
Not possible, because the function uses `this` as parent for QMessageBox and there it can't be const